doc/tesseract.1.asc

```diff
@@ -268,7 +277,7 @@ The engine was developed at Hewlett Packard Laboratories Bristol and at
 Hewlett Packard Co, Greeley Colorado between 1985 and 1994, with some more
 changes made in 1996 to port to Windows, and some C\+\+izing in 1998. A
 lot of the code was written in C, and then some more was written in C\+\+.
-The C\+\+ code makes heavy use of a list system using macros. This predates
+The C++ code makes heavy use of a list system using macros. This predates
```

Comment:
`asciidoc` requires escaping all but the last C++ in a paragraph. Alternatively each `C++` could be written as `$$C++$$`.